tests/test_to_container.py

```diff
@@ -39,32 +39,57 @@ def assert_container_with_primitives(item: Any) -> None:
 
 
 @mark.parametrize(
-    "cfg,ex_false,ex_true",
+    "src,ex_dict,ex_dict_config,key",
     [
         param(
             {"user": User(age=7, name="Bond")},
             {"user": {"name": "Bond", "age": 7}},
-            {"user": User(age=7, name="Bond")},
+            {"user": OmegaConf.structured(User(age=7, name="Bond"))},
```

Comment:
These two are identical because User is being converted to DictConfig anyway.
```
{"user": User(age=7, name="Bond")},
{"user": OmegaConf.structured(User(age=7, name="Bond"))},
```

Reply:
Ok, I changed it back in https://github.com/omry/omegaconf/pull/552/commits/dbefee2425dde48f0a446b85159da9e7c50fcd16.